README.md

```diff
@@ -87,9 +87,11 @@ spark-submit \
 	--class org.hammerlab.guacamole.Guacamole \
 	--verbose \
 	/path/to/target/guacamole-with-dependencies-<x.y.z>.jar \
-	germline-threshold \
-        --reads hdfs:///path/to/reads.bam \
-        --out hdfs:///path/to/result.vcf
+	somatic-joint \
+    		src/test/resources/synth1.normal.100k-200k.withmd.bam \
```

Comment:
I'd put a path or path placeholder that doesn't suggest it's reading from the test `.bam`s in this repo, and maybe add a comment here/above explaining that in `--master yarn` mode paths resolve to HDFS by default.

When to use `file://`, `hdfs://`, or neither proves consistently tricky for newcomers so it would be nice to have a note about that somewhere, but this doesn't have to be the place for it or time to add it.